### ISSUE TITLE
Can't import component from forma-36-react-components

### ISSUE THREAD

**Issue by mealeyst** [contentful/forma-36#117]
Greetings!

I was trying to get started using the `forma-36-react-component` in a `create-react-app` v2.1.5, yet when I try to import a component as is shown in the tutorial, I am getting a message that the components are not being exported:

```
Failed to compile
./src/index.tsx
Attempted import error: 'Button' is not exported from '@contentful/forma-36-react-components'.
```

My index.tsx is as follows:
```
import React from 'react';
import ReactDOM from 'react-dom';
import '@contentful/forma-36-react-components/dist/styles.css';
import { Button } from '@contentful/forma-36-react-components';

class App extends React.Component {

  render() {
    return (
      <Button buttonType="primary" onClick={() => {
        console.log('You clicked on Forma36 button');
      }}>
        Click on me
      </Button>
    );
  }
}
```

I was trying to use the latest version 2.9.2 of the forma-36-react-components.

**Comment by m10l** [comment]
Hi, we'll look into a fix. As a temporary workaround you could do the following with your import:

```jsx
import { Input } from '@contentful/forma-36-react-components/dist/index.js';
```

**Comment by suevalov** [comment]
@mealeyst It's fixed in `2.9.3`. Sorry for inconvenience and thanks for heads up!
